vespalib/src/vespa/vespalib/util/shared_string_repo.cpp

```diff
@@ -6,6 +6,18 @@ namespace vespalib {
 
 SharedStringRepo::Partition::~Partition() = default;
 
+void
+SharedStringRepo::Partition::find_leaked_entries(size_t my_idx) const
+{
+    for (size_t i = 0; i < _entries.size(); ++i) {
+        if (!_entries[i].is_free()) {
+            size_t id = (((i << PART_BITS) | my_idx) + 1);
+            fprintf(stderr, "WARNING: shared_string_repo: leaked string id: %zu ('%s')\n",
```

Comment:
Prefer using LOG() instead.

Reply:
Can we guarantee that the logging pipeline is still intact at this point?

Reply:
@arnej27959 / @baldersheim / @toregge : Any comments on that?

Reply:
I am not sure what c++ says abut destruction order of static versus global objects